python-package/xgboost/spark/core.py

```diff
@@ -583,9 +636,7 @@ def _train_booster(pandas_df_iter):
             evals_result = {}
             with RabitContext(_rabit_args, context):
                 dtrain, dvalid = create_dmatrix_from_partitions(
-                    pandas_df_iter,
-                    None,
-                    dmatrix_kwargs,
+                    pandas_df_iter, features_cols_names, dmatrix_kwargs
```

Comment:
What if features_cols_names conflicts with label / weight / base_margin column name ?

Reply:
Thx, I tried, it worked, no exception happened.